Code/Framework/AzToolsFramework/Tests/EditorTransformComponentSelectionTests.cpp

```diff
@@ -1751,6 +1755,58 @@ namespace UnitTest
                 AZ::Transform::CreateTranslation(EditorTransformComponentSelectionViewportPickingFixture::Entity2WorldTranslation),
                 AZ::Transform::CreateTranslation(EditorTransformComponentSelectionViewportPickingFixture::Entity3WorldTranslation) }));
 
+    class EditorTransformComponentSelectionScaleManipulatorInteractionTestFixture
```

Comment:
If you're not overriding any logic could you also just do using Editor... = Editor...?

Reply:
Yeah I think I started with a custom override for Setup/Teardown and then realized I didn't need them, I can simplify this